src/CMakeLists.txt

```diff
@@ -28,6 +28,8 @@ option(CMAKE_POSITION_INDEPENDENT_CODE "Position independent code" ON)
 
 include(cmake/compiler_flags.cmake)
 
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
+
 find_package(CURL REQUIRED CONFIG)
```

Comment:
Surprised that it works with `CONFIG` on all platforms :thinking:

Reply:
Seems like it doesn't :sweat_smile:. Let me try to just remove that and restart the CI.